### DISCUSSION TITLE
Reusable fields or preset fields

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17160] [Feature Requests]
It would be nice if we could reuse fields for our collections

Example:
I have meta fields (for seo) like `meta_title`, `meta_description` and `meta_image`, which I want to reuse on multiple collections. At the moment I have to create them over and over again. It would save so much time and would be so much more cleaner, if we could create “preset fields” (or whatever name), that we then can insert in every collection and if we need to change something in the field like the `Soft Limit` or the `Display` we change it once – and every instance of the field updates automatically.

**Comment by d1rOn** [comment]
Hi. I use another collection for this purpose.
It's quite convinient, and this collection appears in the API response as detached object.
To make the fields look like native ones, I suggest making a ["Flat interface"](17023).

**Reply by ghost** [reply]
@d1rOn At the moment, I am not convinced of using a drawer for this purpose, but I really like your Idea of a Flat Interface.

**Comment by ghost** [comment]
This is a duplicate of https://github.com/directus/directus/discussions/17023

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
